Crap, a test failed on AppVeyor:

```
FAIL: test_sendfile_close_peer_in_the_middle_of_receiving (test.test_asyncio.test_events.ProactorEventLoopTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\cpython\lib\test\test_asyncio\test_events.py", line 2506, in test_sendfile_close_peer_in_the_middle_of_receiving
    self.file.tell())
AssertionError: False is not true : 65536
```

--- a/Lib/test/test_asyncio/test_events.py
+++ b/Lib/test/test_asyncio/test_events.py
@@ -2113,10 +2113,10 @@ class SubprocessTestsMixin:
 
 class SendfileBase:
 
-    DATA = b"SendfileBaseData" * (1024 * 4)  # 64 KiB
+    DATA = b"SendfileBaseData" * (1024 * 8)  # 128 KiB
 
     # Reduce socket buffer size to test on relative small data sets.
-    BUF_SIZE = 1024   # 1 KiB
+    BUF_SIZE = 4 * 1024   # 4 KiB
 
     @classmethod
     def setUpClass(cls):

